ads/inabox/inabox-messaging-host.js

```diff
@@ -129,6 +129,15 @@ export class InaboxMessagingHost {
       return false;
     }
 
+    const allowedTypes =
+        this.iframeMap_[request['sentinel']].iframe.dataset['ampAllowed'];
+    // having no whitelist is legacy behavior so assume all types are allowed
+    if (allowedTypes &&
+        !allowedTypes.split(/ *, */).includes(request['type'])) {
```

Comment:
I assume commenting out of split here was unintentional and/or can be removed?

Reply:
If you're talking about the "/ *, */" part, that's actually a regex that matches commas with any number of spaces before or after it as a separator.

Reply:
Ah, I suggest using \s for whitespace character as its both easier to read here and allows for other whitespace characters.  Or you could save yourself a split operation and then includes (two O(N) operation) and do:

if (allowedTypes && new RegExp(`(^|,)\\s*${allowedTypes['type']}\\s*($|,)`).test(allowedTypes)) {

Although this could allow someone to subvert the regexp but since the type is checked later essentially against a whitelist, it should be fine.  I'd be curious to know how often inabox host script receives messages.

Last option would be to cache the result of calling split on AdFrame but that means if the publisher changes the whitelist, you wouldn't see it.

Reply:
Thanks for the tip. I changed it as you suggested.